package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@influxdata/clockface",
-  "version": "4.7.4",
+  "version": "4.8.0",
```

Comment:
A semvar point - I think this would need to be 5.0.0 since it breaks backwards compatibility by removing previously exported icons

Reply:
Roger that